src/js/models/maps/Map.js

```diff
@@ -54,9 +64,11 @@ define([
        * @property {Boolean} [showHomeButton=true] - Whether or not to show the
        * home button in the toolbar.
        * @property {Boolean} [showViewfinder=false] - Whether or not to show the
-       * viefinder UI and viewfinder button in the toolbar. The ViewfinderView
+       * viewfinder UI and viewfinder button in the toolbar. The ViewfinderView
        * requires a Google Maps API key present in the AppModel. In order to 
        * work properly the Geocoding API and Places API must be enabled. 
+       * @property {Boolean} [showZoomPresets=false] - Whether or not to show
```

Comment:
It seems like adjusting the configuration is not completely straightforward, we saw this the other day in the demo site where layer configurations started to affect the ability to render the whole site. Similar to the way we have layers and showLayerList, I think it makes sense to have a simple boolean toggle

Reply:
are you referring to when the missing bracket caused a breakage? that would've still broken even with a boolean toggle no?

Reply:
Yes, that's the instance. If we expect to be able to use these flags as feature flags, then there should be a way that is less error prone to turn them off. I wouldn't want to be in a production-affecting situation where we needed to remove some configuration that then caused the entire production Permafrost portal to be unusable

Reply:
If we need to turn the feature off in production, then we flip a boolean toggle rather than deleting a large object, this seems less error prone and less likely to result in a syntax error in the portal document

Reply:
I think of this as a stand in for feature flags, but I'll remove since it seems you both agree it's not necessary.